src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1810,13 +1810,18 @@ def list_function_app_runtimes(cmd, os_type=None):
     return {WINDOWS_OS_NAME: windows_stacks, LINUX_OS_NAME: linux_stacks}
 
 
-def list_flex_function_app_runtimes(cmd, location, runtime):
-    runtime_helper = _FlexFunctionAppStackRuntimeHelper(cmd, location, runtime)
-    runtimes = [r for r in runtime_helper.stacks if runtime == r.name]
-    if not runtimes:
-        raise ValidationError("Runtime '{}' not supported for function apps on the Flex Consumption plan."
-                              .format(runtime))
-    return runtimes
+def list_flex_function_app_runtimes(cmd, location, runtime, ignore_error=False):
+    try:
+        runtime_helper = _FlexFunctionAppStackRuntimeHelper(cmd, location, runtime)
+        runtimes = [r for r in runtime_helper.stacks if runtime == r.name]
+        if not runtimes:
+            raise ValidationError("Runtime '{}' not supported for function apps on the Flex Consumption plan."
+                                  .format(runtime))
+        return runtime_helper.stacks
```

Comment:
This is a bit confusing in that we consume the runtime param but when it is passed, we do not just return info specific to the runtime. I thought the ticket was to just modify the `az functionapp list-flexconsumption-locations` command? This would modify the `az functionapp list-flexconsumption-runtimes` command.

Reply:
Addressed it.